src/action/common/configure_init_service.rs

```diff
@@ -471,6 +458,28 @@ impl Action for ConfigureInitService {
                 )
                 .await
                 .map_err(Self::error)?;
+
+                for attempt in 1..100 {
+                    tracing::trace!(attempt, "Checking to see if the daemon is down yet");
+                    if execute_command(
+                        Command::new("launchctl").process_group(0).arg("print").arg(
+                            [
+                                DARWIN_LAUNCHD_DOMAIN,
+                                self.service_name
+                                    .as_ref()
+                                    .expect("service_name should be defined for launchd"),
+                            ]
+                            .join("/"),
+                        ),
+                    )
+                    .await
+                    .is_err()
+                    {
+                        tracing::trace!(attempt, "Daemon is down");
+                        tokio::time::sleep(tokio::time::Duration::from_millis(100)).await;
+                        break;
```

Comment:
This looks wrong lol

Why are we sleeping 100ms if it is down, instead of sleeping 100ms between attempts to check if it's down...?